### ISSUE TITLE
[Visual Bug] Metrics

### ISSUE THREAD

**Issue by paintballlaws** [dysath/seat-srp#78]
When selecting different tabs for ships/pilots and ships/summary the second tab stays highlighted even thou the other tab is selected.
![chrome_XYJ9AUM7Xz](https://user-images.githubusercontent.com/2366416/136511359-e787032e-ed6a-4962-88ad-7298b890fcd7.png)
![chrome_Ov2b3GVx4J](https://user-images.githubusercontent.com/2366416/136511393-faa7dc7d-a396-4c16-80a1-c4a1a8f50fae.png)

**Comment by Crypta-Eve** [comment]
Yeah not sure what is going on with this one... Not going to fix it for SeAT 4 sorry, but will leave the issue open
